test/e2e-test_positive_verification.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#######################################################################
+assert ()                 #  If condition false,
```

Comment:
You only have to define this method once and can import that in all test cases. Check https://github.com/gnosis/dex-zksnarks/blob/master/ZoKrates/test/run_tests.sh#L3